package.json

```diff
@@ -29,7 +29,6 @@
   "dependencies": {
     "chalk": "^3.0.0",
     "chokidar": "^3.3.1",
-    "eth-ens-namehash": "^2.0.8",
```

Comment:
Haha, I wonder if we should regenerate the package-lock (or even just move entirely to yarn...).

I would suspect more dependencies to be dropped from the lock file if we've removed this dependency.

Reply:
Move to yarn on #112